projects/v2/src/app/components/legend/legend.component.html

```diff
@@ -16,7 +16,14 @@
               'border': l.style === 'stroke' ? '3px solid ' + l.color : 'none',
               'border-style': l.style === 'stroke' ? 'dotted' : 'none'
             }"
-            class="mr-2 circle"
+            [ngClass]="{
+              'circle' : l.name === 'Gene Biomarkers' || !l.bmType,
```

Comment:
I see you are assigning null/undefined bmTypes to 'circle'. Is there another shape to use? One that looks vaguely like an error?

Reply:
if the object doesnt have btype property that means it is either an AsS or CT.